doc/src/tutorials/physics/mechanics/rotating_disc_block.svg

```diff
@@ -0,0 +1,55 @@
+    <path fill="#000000" fill-opacity="0.0" d="m573.7228 412.4954l-37.35437 11.023621" fill-rule="evenodd"/>
+    <path stroke="#000000" stroke-width="2.6666666666666665" stroke-miterlimit="800.0" stroke-linecap="butt" d="m573.7228 412.4954l-37.35437 11.023621" fill-rule="evenodd"/>
+    <path fill="#000000" fill-opacity="0.0" d="m848.1905 295.15445l0 -96.18898" fill-rule="evenodd"/>
+    <path stroke="#000000" stroke-width="2.6666666666666665" stroke-miterlimit="800.0" stroke-linecap="butt" d="m848.1904 295.15445l0 -80.18898" fill-rule="evenodd"/>
+    <path fill="#000000" stroke="#000000" stroke-width="2.6666666666666665" stroke-linecap="butt" d="m852.5951 214.96547l-4.404663 -12.101593l-4.404602 12.101593z" fill-rule="evenodd"/>
+    <path fill="#000000" fill-opacity="0.0" d="m848.1905 294.68732l95.055115 62.015747" fill-rule="evenodd"/>
+    <path stroke="#000000" stroke-width="2.6666666666666665" stroke-miterlimit="800.0" stroke-linecap="butt" d="m848.1904 294.68732l81.65491 53.273163" fill-rule="evenodd"/>
+    <path fill="#000000" stroke="#000000" stroke-width="2.6666666666666665" stroke-linecap="butt" d="m927.43854 351.6494l12.542053 2.9234924l-7.7285767 -10.301392z" fill-rule="evenodd"/>
+    <path fill="#000000" fill-opacity="0.0" d="m848.1903 295.15445l-58.141724 26.362213" fill-rule="evenodd"/>
+    <path stroke="#000000" stroke-width="2.6666666666666665" stroke-miterlimit="800.0" stroke-linecap="butt" d="m848.1903 295.15445l-43.56964 19.755035" fill-rule="evenodd"/>
+    <path fill="#000000" stroke="#000000" stroke-width="2.6666666666666665" stroke-linecap="butt" d="m802.80176 310.89795l-9.202698 9.00885l12.840454 -0.9857788z" fill-rule="evenodd"/>
+    <g transform="matrix(0.9568503937007874 0.0 0.0 0.7775425369585419 806.0367026246719 158.5108997375328)"><clipPath id="p1.1">
+    <path d="m1.1368684E-13 0l100.0 0l0 40.95277l-100.0 0z" clip-rule="evenodd"/>
+    </clipPath>
+    <image clip-path="url(#p1.1)" fill="#000" width="100.0" height="43.0" x="0.0" y="0.0" preserveAspectRatio="none" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAAArCAYAAACO7C3tAAACS0lEQVR4Xu3ZN4gXQRgF8GcEA4IBMWClGEAEbQQLwUoQURTBTjEWgoUgWAja6IGhEiwEbQRRUKy0sJE/FmIhFmJhYS5EMGHAwvy+nZn7Zodbz2L339z7wePYeXNXzLJh9gARERERERERERERERkhFjKTy8FoBrOUmVgW0q5xzDHmBfOH+czszfrVzP3YWb4zB7JeWmRXw234Yqc8jf0O5ucQ/RdmQpwjLbqOsMCXmFnMo3j8m9kefz5k1jDLmNext2yGtG4lc5IZHY/zW9NX5gEzPXbmStbvz8alI5/gC/6RmVOvqysp9ZuKTlo2H/XnxLZ6XXkM7+cVnbTMTkBa7CdFZ+zWlfpXRScdOAdfcHsNLtktKvUXi046kN6wLMuLzpyB97uKTlo2FeEV1xbb9hhj6nXFXn/TCbHnzVBsJ39wmGwZnC2N1sEX2zaKpSnML4T+TdHlTsH/ju3o38fkm8u7g7Ol0XH4gg0UnVkF72/GsbHMCWZRmkR3EOYcgl9l9ur8LI6/ZObGcfmHHnzBN9Srynp432OmMdfi8Xmfhg8Iu//E5qVn0ztmcdZJA/u4+A2+4DPrdcW+9P6Az0m5zIzP5uUmMfcQ5tmu374KyH9YAl9gu7U0OY3wXLB5z5kjzKjaDGcn6RbCXPudtfVa2jIbw9927LuYXTl2MuxFYGu9ln7L9yv7ik767DD8ZBzNxhcg7O5XZGPSsd3wk3G26K7Gcds4Sh9shG/+3iKcnJ3MHuZCHLcvAOn/LtKxG/Croym9NFlERERERoq/CdyuuhTlTpsAAAAASUVORK5CYII="/>
```

Comment:
Can you clean up this SVG? It seems that it contains references to images. Personally, I like to use inkscape for this purpose.
And can you make it dark theme compatible refer to #24695

Reply:
I tried to delete `<image>` tags that have a reference (`xlink:href`), but it affects the images (some parts of the image are removed). I can redraw the diagram in Inkscape.

And I checked that the image displays correctly in dark mode.

<img width="770" alt="Screenshot 2024-08-12 at 19 05 45" src="https://github.com/user-attachments/assets/d9aa3093-f974-426c-9d73-30b10ea632f1">

Reply:
I installed Inkscape and redrew the schematic, but the arrows in the image disappear when I download it as SVG for some reason.

<img width="794" alt="Screenshot 2024-08-14 at 00 02 39" src="https://github.com/user-attachments/assets/3d2d1623-c326-49a7-ac08-9bf666d8a816">

<img width="576" alt="Screenshot 2024-08-14 at 00 07 04" src="https://github.com/user-attachments/assets/7470819d-688d-4c3c-883c-4ca8bd261741">

Reply:
Nice top view, that makes the image/system easier to understand. I don't remember the exact reason that this problem may occur (I did see it before). An option is to try File-> clean document (I believe it was called something like that) and save it as plain svg. Another option is to just open one of the SVGs of a joint and copy an arrow from there and update the color.

Reply:
I directly edited the SVG file code, changing `fill-opacity:0` to `fill-opacity:1` and `stroke:context-stroke` to `stroke:#000000`. I also set the background color to white using `<rect width="100%" height="100%" fill="white"/>`. So now, it correctly displays in dark mode.

Reply:
Small question: the coordinate systems A and N do not seem to line up for q_1 = 0, or so I understand the sketches. Any reason for this?
Thanks!

Reply:
Hi Peter, thanks for your interest in this PR! N is the inertia frame and A is the rotating frame, and frame A is used to define the position and velocity of q2 and q3. q1 won't be 0 at any time since I assume the initial velocities of q1, q2, and q3 are not zero, allowing us to use `CoulombKineticFriction` actuator, which is designed for kinetic friction.

Reply:
What I meant was only this:
If you lign up N_x and A_x then N_y and A_y point in different directions. Was this intentional?
(N_z and A_z also have to point in opposite directions, else N or A is not right handed as per your sketch)

Reply:
![IMG_8726](https://github.com/user-attachments/assets/6b496495-b774-4868-b380-7f633fe806f0)

This is what I intend for N and A, so Nx and Ny don't line up with Ax and Ay, but Nz lines up with Az

Reply:
Sorry I drew a wrong picture for top view, this is the correct one
![IMG_8727](https://github.com/user-attachments/assets/d4acf731-3462-46db-99d1-2bc75bb014d0)